unlock-app/src/services/walletService.js

```diff
@@ -23,7 +28,7 @@ export const keyId = (lock, owner) => [lock, owner].join('-')
  * actually retrieving the data from the chain/smart contracts
  */
 export default class WalletService extends EventEmitter {
-  constructor({ providers, unlockAddress } = configure()) {
```

Comment:
Is there context around why we are no longer calling configure here?

Reply:
configure is unlock-app/paywall only, and unlock-js aims to be agnostic about this is my understanding

Reply:
@cellog  said it all ;)